app/index.js

```diff
@@ -1,37 +1,63 @@
 import React, {Component} from 'react';
+import{Linking} from 'react-native';

```

Comment:
u need coherent spacing, set it up correctely plz, if not automated, automate it